lib/allure_report_publisher/lib/providers/gitlab.rb

```diff
@@ -58,28 +58,67 @@ def update_pr_description
         )
       end
 
+      # rubocop:disable Metrics/CyclomaticComplexity
+      # rubocop:disable Metrics/PerceivedComplexity
```

Comment:
I could have moved the additional logic for alert_comment in a different method but then we go above the 20 method threshold and code climate would complain